src/lib/components/about/about-us.svelte

```diff
@@ -1,6 +1,6 @@
 <section>
   <div class="container">
-    <img src="assets/about-illustration.svg" alt="acmCSUF badges" />
+    <img src="assets/png/GroupPhoto.png" alt="acmCSUF badges" />
```

Comment:
update the alt text since it no longer reflects acm badges

Reply:
also see other comment about naming scheme: `group-photo.png`

Reply:
changed it to ACM family :)

Reply:
oh also can you convert this file to either a jpeg or a webp? pngs are big and load slow

Reply:
one more thing: if the with-shadows folder isn't used anymore can you remove those too?

Reply:
> one more thing: if the with-shadows folder isn't used anymore can you remove those too?

I'll leave that folder in there cause it cause the Node Buds drop shadow logo

Reply:
> oh also can you convert this file to either a jpeg or a webp? pngs are big and load slow

Alright I changed the group photo in the About Us to a jpeg!!